operator/src/main/java/oracle/kubernetes/weblogic/domain/model/DomainResource.java

```diff
@@ -428,6 +431,15 @@ public String getOpssWalletFileSecret() {
     return spec.getOpssWalletFileSecret();
   }
 
+  /**
+   * Reference to secret opss key passphrase for model in image.
+   *
+   * @return opss key passphrase
+   */
+  public String getModelOpssWalletPasswordSecret() {
```

Comment:
There is already a similar method getOpssWalletPasswordSecret() that will return either secret for model in image or psi case,  may be we should consolidate them?